### DISCUSSION TITLE
Text-based UI

### DISCUSSION THREAD

**Discussion by brandonkal** [karakeep-app/karakeep#239] [Ideas]
Will hoarder consider a text-based user-interface like Google Search? The card and image-heavy tiling layout cannot contain as much information as a simple text list with description snippets which makes finding a saved page more difficult. This is form over function

**Comment by MohamedBassem** [comment]
Have you checked the list layout? Is this good enough or do you need something more dense?

**Reply by brandonkal** [reply]
I do not see it here https://try.hoarder.app/dashboard/search but ideally something dense. I use https://historio.us/ but looking to migrate to something else as the URL data fetcher doesn’t always return indexable content.

**Reply by MohamedBassem** [reply]
I'm on the phone now, but if you click on the three dots at the bottom left of the webpage, you'll be able to change the layout.

**Reply by brandonkal** [reply]
I see. On hoarder I can see 5 articles on screen. On historious I can see 12. Hoarder is also missing the description entry
